Source/Reporters/FileAppendingReporter.jsl

```diff
@@ -51,6 +52,10 @@ Define Class(
 	
 	// Function: add success
 	add success = Method( {label, test expr, description, payload=Empty()},
+		if(this:write successes,
+			success = Eval Insert( "\!NSUCCESS: ^Name Expr(test expr)^ ^description^ (^label^)" );
```

Comment:
Should we be using something like the `shortDescription` above? This description can be very large in some cases (e.g. large matrices).

Reply:
I think you're right - it would get messy. I was debating it because failures show the full description as well but successes are less important anyway. I made the change.